docs/end-user-topics/dois.rst

```diff
@@ -69,10 +69,18 @@ Automatic Dockstore DOI Generation
 .. note::
     This feature is currently only available on request. Please contact the Dockstore team at `discuss.dockstore.org <https://discuss.dockstore.org/t/opening-helpdesk-tickets/1506>`__ if you want to participate.
 
-* Automatically mints DOIs for all Git tags pushed to GitHub repositories that have a corresponding published entry in Dockstore
-* The DOIs are minted in the Zenodo Dockstore community
-* No Zenodo account is required to mint DOIs
-* A linked Zenodo account is required to edit DOIs
+Dockstore will automatically mint a DOI for a version that corresponds to a Git tag, if:
+
+* The entry is published on Dockstore
```

Comment:
Do you mean if the entry is already published, or the act of publishing? I'd read it as the former, but then looking at line 76, I think you mean the act of publishing. If so, I suggest something like "Publishing an entry on Dockstore".

Reply:
> Do you mean if the entry is already published, or the act of publishing? I'd read it as the former, but then looking at line 76, I think you mean the act of publishing. If so, I suggest something like "Publishing an entry on Dockstore".

Both meanings. I changed the last bullet to clarify the two cases a little better.
Might not describe all of the scenarios 100% precisely, but short and readable and more future proof than trying to quantify the retroactive DOI generation that we'll probably do later but don't right now...